src/axom/primal/tests/primal_clip.cpp

```diff
@@ -1260,6 +1260,36 @@ TEST(primal_clip, tet_tet_clip_split)
   EXPECT_NEAR(0.3333, tet_volumes, EPS);
 }
 
+TEST(primal_clip, tet_tet_clip_special_case_1)
+{
+  using namespace Primal3D;
+  constexpr double EPS = 1e-10;
+  constexpr bool CHECK_SIGN = true;
+
+  // Tets do not intersect, but share a coplanar face
```

Comment:
```suggestion
  // Tets do not intersect, but share a face
```

nit: faces have 3 vertices, so are planar by definition. so a shared face would have to be in the same plane.